composer.lock

```diff
@@ -6230,6 +6230,73 @@
         }
     ],
     "packages-dev": [
+        {
+            "name": "masterminds/html5",
```

Comment:
A lot of dependencies are added in composer.lock, but not in the composer file. Also these dependencies are unrelated to your actual implementation, please revert

Reply:
Done.